controller/Previewer.php

```
@@ -223,54 +221,32 @@ public function submitItem(): void
     public function getTokens(): void
     {
         try {
-            $this->setSuccessJsonResponse($this->getAccessTokenService()->fetchTokens());
-            $this->logPreviewViewEventFromTokenRequest($this->getPsrRequest()->getQueryParams());
+            $this->setSuccessJsonResponse(
+                $this->getAccessTokenService()->fetchTokens(
```

PHPStan (2.2.7)

[error] Method oat\tao\model\accessControl\Service\AccessTokenService::fetchTokens() invoked with 1 parameter, 0 required.


(arguments.count)